README.md

```diff
@@ -44,20 +44,56 @@ If you want to install this package as dependency, you can install it from this
 npm install --save https://github.com/HatScripts/circle-flags
 ```
 
+### Custom builds
+
+You can create a customized build by running the included script:
+
+```
+$ npm install
+$ CORNER_RADIUS=25% npm run build
+```
+
+This will generate flags with rounded corners instead of the default circle. The files will appear in the `build/` directory.
+
+To build only selected icons, pass their codes as arguments:
+
+```
+$ CORNER_RADIUS=25% npm run build gb us ca
+```
+
+The following environment variables are currently recognized:
+
+| Variable        | Default value  |
+| --------------- | -------------- |
+| `SRC_DIR`       | `src/`         |
+| `BUILD_DIR`     | `build/`       |
+| `CORNER_RADIUS` | `50%` (circle) |
+
 ## Contributing
 
-To contribute, you need to have [svgo](https://github.com/svg/svgo) installed
-(version 1.2.0 or newer).
+To contribute, clone the repository and install the dependencies:
+
+```
+$ git clone https://github.com/HatScripts/circle-flags.git
+$ cd circle-flags/
+$ npm install
+```
+
+Edit the relevant SVG files under `src/`, then run the build script on them, e.g.:
 
-First, edit the relevant SVG files in the `flags/` directory.
+```
+$ npm run build us ru cn
+```
 
-Then run `svgo` to optimize the SVG files:
+The files will appear under `build/`. Check if everything looks good, then move the files into `flags/`.
```

Comment:
Wouldn't it be better to have the default build command output to `flags/` directly? Since this is a git directory, it's not like overwriting the files would cause any irreversible damage. Unless you think this is useful for comparing the original and the edited built files?

Reply:
Yeah, I used it that way myself, to compare the file sizes against the default build, but maybe an extra "staging area" like that is redundant. I thought perhaps this could be useful to users who haven't decided yet which variant they want to use, so they could compare the builds side by side in their file manager. I'd like to see what others think before changing it!